docs/developer_manual/services/adding_a_service_updater.md

```diff
@@ -1,85 +1,210 @@
+            # The contents in this directory will then be used by prepare_output_directory().
+
+            # Organize files by source
+            dest_dir = os.path.join(self.latest_updates_dir, source)
 
-Let's say the following code is written in `update_server.py` in the root of your service directory:
-
-```python
-from assemblyline.odm.models.signature import Signature
-from assemblyline_v4_service.updater.updater import ServiceUpdater
-
-class SampleUpdateServer(ServiceUpdater):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-    def import_update(self, files_sha256, client, source, default_classification): -> None
-        # Purpose:  Used to import a set of signatures from a source into Assemblyline for signature management
-        # Inputs:
-        #   files_sha256:           A list of tuples containing file paths and their respective sha256
-        #   client:                 An Assemblyline client used to interact with the API on behalf of a service account
-        #   source:                 The name of the source
-        #   default_classification: The default classification given to a signature if none is provided
-
-        signatures = []
-        for file, _ in files_sha256:
-            # Iterate over all the files retrieved by source and upload them as signatures in Assemblyline
-            with open(file, 'r') as fh:
-                signatures.append(Signature(dict(
-                    classification = default_classification,
-                    data = fh.read(),
-                    name = f'sample_signature{len(signatures)}',
-                    source = source_name,
-                    status = 'DEPLOYED',
-                    type = 'sample'
-                )))
-        client.signatures.add_update_many(signatures)
-        self.log.info(f'Successfully imported {len(signatures)} signatures')
-        return
-
-    def is_valid(self, file_path) -> bool:
-        # Purpose:  Used to determine if the file associated is 'valid' to be processed as a signature
-        # Inputs:
-        #   file_path:  Path to a signature file from an external source
-        return super().is_valid(file_path) #Returns true always
-
-if __name__ == '__main__':
-    with SampleUpdateServer(default_pattern="*.json") as server:
-        server.serve_forever()
-```
+            # For every file in this source, move to latest updates directory in a subdirectory labelled by source.
+            for file, _ in files_sha256:
+              shutil.move(file, dest_dir)
+            self.log.info(f"Finished moving {len(files_sha256)} files for {source} to {self.latest_updates_dir}")
+            return
+
+        def is_valid(self, file_path) -> bool:
+            # Purpose:  Used to determine if the file associated is 'valid' to be processed as a signature
+            # Inputs:
+            #   file_path:  Path to a signature file from an external source
```

Comment:
Would adding another comment next to this one
#  This is the default implementation : 
Be a good idea? So it shows that they don't need to re-implement it if they want this default behaviour?